README.md

```diff
@@ -69,7 +86,19 @@ AZNFS is supported on following Linux distros:
 	```
 	sudo mount -t aznfs -o vers=3 <account-name>.blob.core.windows.net:/<account-name>/<container-name> /mountpoint
 	```
-- Logs generated from AZNFS can be found in `/opt/microsoft/aznfs/data/aznfs.log`.
+- Mount the Azure File NFS share using following command:
```

Comment:
I guess you should create different subsections for NFSv3 Blob and NFSv4 files under usage instructions for better readability.